### PR TITLE
Add Compatiblity

### PR DESCRIPTION
**Pull request by jerzzhang**

### CODE REVIEW

File: smartcar/smartcar.py [hunk 1]
```diff
@@ -161,3 +161,21 @@ def exchange_refresh_token(self, refresh_token):
...
+
+        Args:
+            vin (str): the VIN of the vehicle
+
+        Returns:
+            boolean: true if the vehicle is compatible
+
+        """
+        method = 'GET'
+        url = const.API_URL
```
Comment (gurpreetatwal) [inline smartcar/smartcar.py]: missing `compatibility` endpoint

does not work as is

File: tests/test_smartcar.py [hunk 2]
```diff
@@ -168,6 +168,17 @@ def test_exchange_token(self):
         self.assertEqual(request().headers['Content-Type'], 'application/x-www-form-urlencoded')
         self.assertEqual(request().body, urlencode(body))
 
+    @responses.activate
+    def test_compatibility(self):
+        query = {
+            'vin': 'vin,'
+        }
+        responses.add('GET', smartcar.const.API_URL, json={
```
Comment (gurpreetatwal) [inline tests/test_smartcar.py]: url missing `/compatiblity`

File: smartcar/__init__.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-__version__ = '1.0.13'
+__version__ = '1.0.14'
```
Comment (gurpreetatwal) [inline smartcar/__init__.py]: thoughts on doing '1.1.0` since this is a new feature?

File: README.md [hunk 4]
```diff
@@ -175,6 +175,20 @@ Exchange a refresh token for a new access dictionary
 | Dictionary.`refresh_expiration` | A datetime of the expiration of the refresh_token |
 | Dictionary.`token_type`         | Always set to  Bearer . Token type is used in forming the Authorization header used by the Smartcar API in the following step. |
 
+### `compatibility(vin)`
```
Comment (sankethkatta) [inline README.md]:
```suggestion
### `is_compatible(vin)`
```

File: smartcar/smartcar.py [hunk 5]
```diff
@@ -161,3 +161,20 @@ def exchange_refresh_token(self, refresh_token):
...
+
+        Returns:
+            boolean: true if the vehicle is compatible
+
+        """
+        method = 'GET'
+        url = const.API_URL + '/compatibility'
+        query = { 'vin': vin }
+        response = requester.call(method, url, params=query, auth=self.auth).json()
+        print(response)
```
Comment (sankethkatta) [inline smartcar/smartcar.py]: remove `print` statement

**Pull request merged**